prime-angular-frontend/src/app/modules/adjudication/shared/components/dated-content-table/dated-content-table.component.html

```diff
@@ -11,11 +11,15 @@
                  class="d-block text-mute font-weight-normal mb-2">
             {{ item.date | formatDate: 'h:mm a' }}
           </small>
-          <mat-chip-list *ngIf="item?.name"
-                         [selectable]="false">
-            <mat-chip selected
+          <mat-chip-list [selectable]="false">
+            <mat-chip *ngIf="item?.name"
+                      selected
                       color="primary"
                       [disableRipple]="true">{{ item.name | replace: '@idir': '' }}</mat-chip>
+            <mat-chip *ngIf="item?.partyName"
```

Comment:
So first off, when we add something like this to the template that references party name and don't reference it in the controller and the default DatedContent it kind of becomes a hidden feature. We probably want to avoid hidden features especially on generic components like this.

This leads into my second comment about i'm not sure if this is how we want to display this data. This is because of the following:

- It removes from the importance of the adjudicator being labelled on the event
- It will appear on every event and clutters the screen
- I don't think it is that important of information since on any site that hasn't been claimed it will all be the same

What we could do is add a footer to the Dated Content Class and then render the footer below the content and be a little more discreet with the font. Just smaller and maybe in a grey from the palette. That way the footer content is exposed for this component and can be used for other uses.